test/NuGet.Core.Tests/NuGet.Protocol.Core.v3.Tests/HttpSource/HttpRetryHandlerTests.cs

```diff
@@ -141,13 +141,18 @@ public class HttpRetryHandlerTests
                 return new HttpResponseMessage(HttpStatusCode.InternalServerError);
             };
 
+            // Make a timeout that is greater than SmallTimeout (the duration of the request, in 
+            // this case) but less than the total time spent on all attempts, which is MaxTries
+            // times SmallTimeout.
+            var requestTimeout = new TimeSpan((SmallTimeout.Ticks * MaxTries) / 2);
```

Comment:
Took me a while to understand the test scenario here. Have to admit it's a bit confusing.
Please disregard my previous comment.